includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -401,7 +402,14 @@ public function __construct(
 		add_action( 'woocommerce_update_options_payment_gateways_' . $this->id, [ $this, 'process_admin_options' ] );
 		add_action( 'admin_notices', [ $this, 'display_errors' ], 9999 );
 		add_action( 'woocommerce_order_actions', [ $this, 'add_order_actions' ] );
-		add_action( 'woocommerce_order_action_capture_charge', [ $this, 'capture_charge' ] );
+		// TODO: REMOVE THIS BEFORE MERGING. Need to fix this once the UPE issue is fixed.
+		if ( ! get_option( 'woocommerce_order_action_capture_charge_added' ) ) {
+			add_action( 'woocommerce_order_action_capture_charge', [ $this, 'capture_charge' ] );
+			update_option( 'woocommerce_order_action_capture_charge_added', 1 );
+		}
+		add_action( 'shutdown', function() {
+			update_option( 'woocommerce_order_action_capture_charge_added', 0 );
+		} );
```

Comment:
As stated, the above code is to get around the UPE bug where you cannot capture payments without an error.
https://github.com/Automattic/woocommerce-payments/issues/5780